lifetimes/fitters/gamma_gamma_fitter.py

```diff
@@ -51,10 +51,14 @@ def __init__(self, penalizer_coef=0.0):
 
     @staticmethod
     def _negative_log_likelihood(params, frequency, avg_monetary_value,
-                                 penalizer_coef=0):
-        if any(i < 0 for i in params):
-            return np.inf
-
+                                 penalizer_coef=0, q_constraint=True):
+        if q_constraint:
```

Comment:
I think the checks of q-constraint and `any(i < 0 for i in params)` are independent, and can be checked seperately. Ex:
```python
if any(i < 0 for i in params):
    return np.inf
if q_constraint and params[1] < 1:
    return np.inf
```